src/font_loader.h

```diff
@@ -14,6 +14,14 @@
 #include "path_info.h"
 #include "cata_utility.h"
 
+// Ensure that unifont is always loaded as a fallback font to prevent users from shooting themselves in the foot
+static auto ensure_unifont_loaded = []( std::vector<std::string> &font_list )
```

Comment:
This can be a normal function instead of a lambda.

Reply:
Thanks I forgot to change it.